rampwf/utils/cli/testing.py

```diff
@@ -92,7 +93,9 @@ def main(submission, ramp_kit_dir, ramp_data_dir, data_label,
         ]
     else:
         submission = [submission]
-
+    if not os.path.exists("utils"):
```

Comment:
I think we should append to `sys.path` only if `utils` exists, otherwise there is no need to create an empty directory.

Reply:
OK!